extras/jenkins/Linode/Jenkinsfile

```diff
@@ -187,7 +187,7 @@ pipeline {
 
           sh '''
             # Destroy our partial build...
-            $WORKSPACE/pulumi/python/runner -p linode -s marajenklke${BUILD_NUMBER destroy|| true
```

Comment:
Are you sure that you don't want a space after `||`?

Reply:
The shell doesn't care, but I think that for readability we should have it; fixed.